prototypes/FormatPage.hs

```diff
@@ -592,7 +596,7 @@ maxKeySize = diskPageSizeBytes dpgsz - overhead  -- 4052
   where
     overhead =
       (pageSizeBytes . fromJust . calcPageSize dpgsz . PageLogical)
-        [(Key BS.empty, Delete, Just (BlobRef 0 0))]
+        [(Key BS.empty, Insert (Value BS.empty) (Just (BlobRef 0 0)))]
```

Comment:
We can't use `Delete` with a `BlobRef` anymore! That's the representation change this PR makes.

But actually, they should have the same overhead.

Reply:
> But actually, they should have the same overhead.

A blob reference adds 12 bytes of overhead, so it's not the smallest overhead we could get. But, I now understand that we need to use the blob reference in the overhead calculation because the key has to be able to fit in a page with a blob reference. Maybe we should just put that in a note here, since we're not precisely calculating the maximum key size, rather the maximum key size that will *always* fit

Reply:
> Maybe we should just put that in a note here, since we're not precisely calculating the maximum key size, rather the maximum key size that will always fit

Yes, that's right and a fair point that we should say so.